Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3-buster as base
+
+EXPOSE 5000
+ENTRYPOINT root/.local/bin/poetry run flask run --host=0.0.0.0
+
+RUN apt-get update 
+RUN apt-get install -y curl
+RUN curl -sSL https://install.python-poetry.org | python3 -
+
+COPY pyproject.toml .
```

Comment:
I'd definitely recommend also copying the `poetry.lock` here - although `pyproject.toml` specifies your top level dependencies, guaranteeing their sub-dependencies consistently use the same versions from the lock file is generally considered best practice so that we're not getting slightly different packages tomorrow than we do today